src/pkcs11/openssl.c

```diff
@@ -227,6 +227,11 @@ void
 sc_pkcs11_register_openssl_mechanisms(struct sc_pkcs11_card *p11card)
 {
 	sc_pkcs11_mechanism_type_t *mt = NULL;
+/*
+ * Engine support is being dropped in 3.0. OpenSC loads GOST as engine.
```

Comment:
```suggestion
 * Engine support is being deprecated in 3.0. OpenSC loads GOST as engine.
```

Reply:
Done in https://github.com/OpenSC/OpenSC/pull/2506/commits/723f559db0bf6d4e7731afe2b77e973662867b7c